core/schema.py

```diff
@@ -473,6 +476,8 @@ def update_or_create_role(data, user):
                     "validity_from": data['validity_from'],
                 }
             ) for right_id in rights_id]
+        if client_mutation_id:
```

Comment:
@dborowiecki Can you add this to " duplicate_role" function? I think it could be also required for @rstencel because it is also a place where the new object (Role) is created.

Reply:
Added for duplication, nice catch